gitpod-io/gitpod
# [supervisor] run tasks after ide ready

## Description

Run .gitpod.yml tasks after code server is ready

## Related Issue(s)

Fixes #8826

## How to test

1. Start workspace with this https://hw-fix-8826.staging.gitpod-dev.com/#https://github.com/shaal/gitpod-test--port-23000
2. Check if tasks run succeed, and code open README.md file

## Release Notes

```release-note
NONE
```

## Documentation
